packages/components/tests/acceptance/components/hds/link/inline-test.js

```diff
@@ -0,0 +1,20 @@
+import { module, test } from 'qunit';
+import { visit, currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'dummy/tests/helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+
+module('Acceptance | Component | hds/link/inline', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('visiting /components/link/inline', async function (assert) {
+    await visit('/components/link/inline');
+
+    assert.strictEqual(currentURL(), '/components/link/inline');
+  });
```

Comment:
It seems we agreed to remove these tests as we can detect if the page doesn't render from the regression tests.

Reply:
dropped, thanks for catching that 👍